include/deal.II/lac/petsc_block_vector.h

```diff
@@ -43,6 +43,8 @@ namespace PETScWrappers
    * interface, this class handles the actual allocation of vectors and
    * provides functions that are specific to the underlying vector type.
    *
+   * This class is deprecated use PETScWrappers::MPI::BlockVector.
```

Comment:
grammar (missing comma):
+   * This class is deprecated, use PETScWrappers::MPI::BlockVector.